### PR TITLE
Feature/breaking news page

### PR DESCRIPTION
**Pull request by TawKhoury**
**Related issue(s):** #1333 

## Description

**Notes:**

## Checklist

  * [ ] Approved by designer

Usability:
  * [ ] Developer
  * [ ] Project manager

Tests
  
  * [ ] UI snapshot testing 
  * [ ] Unit testing
  * [ ] Integration testing
  * [ ] E2E testing

### CODE REVIEW

File: packages/apps/haaretz.co.il/layouts/BreakingNewsLayout.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  if (window.matchMedia("(min-width: 37.5em)").matches) {
+    var refreshTimer = setTimeout(
+      function refreshWindow() { typeof window != 'undefined' &&  window.location.reload() },
+      ${1000 * 60 * 5}
+    );
+    window.__HTZ.cancelRefresh = function cancelRefresh() {
+      clearTimeout(refreshTimer);
+    };
+  }
+}
```
Comment (TxHawks) [inline packages/apps/haaretz.co.il/layouts/BreakingNewsLayout.js]: The refresh script should only be added in the homepage

File: packages/apps/haaretz.co.il/utils/getBreakingNewsElements.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { dfpTargeting, } from '@haaretz/dfp';
+
+// List views
+
+import {
+  // General Elements
+  ArticleBody,
```
Comment (TxHawks) [inline packages/apps/haaretz.co.il/utils/getBreakingNewsElements.js]: Do we really need `<ArticleBody />` in the breaking news page?

File: packages/apps/haaretz.co.il/utils/getBreakingNewsElements.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  Breadcrumbs,
+  ChangeableElementGroup,
+  ClickTrackerWrapper,
+  Debug,
+  ElementGroup,
+  Error,
+  Footer,
+  GeneralAdSlot,
+  HtmlElement,
```
Comment (TxHawks) [inline packages/apps/haaretz.co.il/utils/getBreakingNewsElements.js]: Or HtmlElement?

File: packages/apps/haaretz.co.il/utils/getBreakingNewsElements.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  withDfpSideEffect,
+  WrappedComments,
+
+  // List views
+  Bender,
+  Farnsworth,
+  Leela,
+  Zoidberg,
+  Clampazzo,
+} from '@haaretz/htz-components';
```
Comment (TxHawks) [inline packages/apps/haaretz.co.il/utils/getBreakingNewsElements.js]: This list can probably be pruned some

File: packages/components/htz-components/src/components/BreakingNews/dayMap.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+const dayMap = {
+  0: 'ראשון',
+  1: 'שני',
+  2: 'שלישי',
+  3: 'רביעי',
+  4: 'חמישי',
+  5: 'שישי',
+  6: 'שבת',
+};
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/BreakingNews/dayMap.js]: This belongs in the theme, not here

Comment (TawKhoury) [reply packages/components/htz-components/src/components/BreakingNews/dayMap.js]: Done, Thanks.

**Pull request merged**